lucuma-schemas/src/clue/resources/lucuma/schemas/ObservationDB.graphql

```diff
@@ -5479,6 +5487,233 @@ enum Flamingos2Fpu {
   LONG_SLIT_8
 }
 
+"""Flamingos2 Read Mode"""
+enum Flamingos2ReadMode {
+  """Flamingos2ReadMode Bright"""
+  BRIGHT
+
+  """Flamingos2ReadMode Medium"""
+  MEDIUM
+
+  """Flamingos2ReadMode Faint"""
+  FAINT
+}
+
+"""Flamingos2 Decker"""
+enum Flamingos2Decker {
+  """Flamingos2Decker Imaging"""
+  IMAGING
+
+  """Flamingos2Decker LongSlit"""
+  LONG_SLIT
+
+  """Flamingos2Decker MOS"""
+  MOS
+}
+
+"""Flamingos2 Readout Mode"""
+enum Flamingos2ReadoutMode {
+  """Flamingos2ReadoutMode Science"""
+  SCIENCE
+
+  """Flamingos2ReadoutMode Engineering"""
+  ENGINEERING
+}
+
+"""Flamingos2 Reads"""
+enum Flamingos2Reads {
+  """Flamingos2Reads Reads 1"""
+  READS_1
+
+  """Flamingos2Reads Reads 3"""
+  READS_3
+
+  """Flamingos2Reads Reads 4"""
+  READS_4
+
+  """Flamingos2Reads Reads 5"""
+  READS_5
+
+  """Flamingos2Reads Reads 6"""
+  READS_6
+
+  """Flamingos2Reads Reads 7"""
+  READS_7
+
+  """Flamingos2Reads Reads 8"""
+  READS_8
+
+  """Flamingos2Reads Reads 9"""
+  READS_9
+
+  """Flamingos2Reads Reads 10"""
+  READS_10
+
+  """Flamingos2Reads Reads 11"""
+  READS_11
+
+  """Flamingos2Reads Reads 12"""
+  READS_12
+
+  """Flamingos2Reads Reads 13"""
+  READS_13
+
+  """Flamingos2Reads Reads 14"""
+  READS_14
+
+  """Flamingos2Reads Reads 15"""
+  READS_15
+
+  """Flamingos2Reads Reads 16"""
+  READS_16
+}
+
+"""Flamingos2 Window Cover"""
+enum Flamingos2WindowCover {
+  """Flamingos2WindowCover Open"""
+  OPEN
+
+  """Flamingos2WindowCover CLOSE"""
+  CLOSE
+}
+
+"""Flamingos2 South Long Slit mode"""
+type Flamingos2LongSlit {
+  """
+  The disperser field must be specified.  It cannot be unset with a null value.
```

Comment:
The comments here read like those you'd expect for an `input` instead of a `type`.

Reply:
These are taken from the odb, we'd need to fix them there